_examples/README.md

```diff
@@ -2,6 +2,31 @@
 
 > All code snippets are run from the root project directory.
 
+## Run Waku node
+
+Running Waku node is a matter of a correct configuration. To enable Waku and JSON-RPC HTTP interface use:
+```shell script
+{
+  "APIModules": "waku",
+  "HTTPEnabled": true,
+  "HTTPHost": "localhost",
+  "HTTPPort": "8545",
+  "WakuConfig": {
+    "Enabled": true
+  }
+}
+```
+
+This command will start a Waku node using the `eth.prod` fleet:
```

Comment:
Is this transparent to non Status people? What alternatives are there to running with `eth.prod` fleet? E.g. adding a node like in geth

Reply:
Currently, there is no way to run status-go without selecting a fleet. Of course, if one wants to use their own nodes, it's possible by providing a proper config overriding properties defined in, for example, https://github.com/status-im/status-go/blob/develop/config/cli/fleet-eth.prod.json.